app/assets/stylesheets/framework_and_overrides.css.scss

```diff
@@ -102,3 +102,52 @@ img {
   height: 25vh;
   background-color: rgba(255,255,255,0.5);
 }
+
+.apartment-info {
+  background-color: #c5499b;
```

Comment:
Color literals like `#c5499b` should only be used in variable declarations; they should be referred to via variable everywhere else.